docs/ad/security-ad.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Anomaly Detection Security
+nav_order: 10
+parent: Anomaly Detection
+has_children: false
+---
+
+# Anomaly detection security
+
+You can use the security plugin with anomaly detection to limit non-admin users to specific actions. For example, you might want some users to only be able to create and view detectors, while others to modify and delete detectors.
+
+Security for anomaly detection works the same as [security for alerting](../../alerting/security/).
+
+## Basic permissions
+
+As an admin user, you can use the security plugin to assign specific permissions to users based on which APIs they need access to. For a list of supported APIs, see [Anomaly Detection API](../api/).
+
+You can also use the following built-in roles that cover most anomaly detection use cases:
+
+- `anomaly_full_access` - All access to anomaly detection APIs
+- `anomaly_read_access` - Read-only access to anomaly detection APIs
+
+If these roles don't meet your needs, mix and match individual anomaly detection [permissions](../../security/access-control/permissions/) to suit your use case. Each action corresponds to an operation in the REST API. For example, the `cluster:admin/opendistro/ad/detector/delete` permission lets you delete detectors.
+
+Sample configuration to fine-tune user access to APIs for all or none of the detectors:
+
+```yaml
+# Allow users to read anomaly detection detectors and its results
+anomaly_read_access:
```

Comment:
Also we should probably add these roles to `docs/security/access-control/users-roles.md`